src/recon_test/test_DataSymmetriesForBins_PET_CartesianGrid.cxx

```diff
@@ -124,14 +124,22 @@ run_tests_2_proj_matrices_1_bin(const ProjMatrixByBin& proj_matrix_no_symm,
 #endif
       elems_no_sym.sort();
       elems_with_sym.sort();
-      if (!check(elems_no_sym == elems_with_sym, "comparing lors"))
+
```

Comment:
Getting an error that these sorts should be preventing... Note the coordinates `{2, 0, 0}` and `{2, 1, 0}` appear twice, once for each `elems`. Note, these results are with a scanner with 0 phi offset.
```
Error. Comparing symmetry LORs
	bin: 	segment = -3, 	axial pos 0, 	view = 0, 	tangential_pos_num = 0, 	timing position index = -1
SymBin :	segment = -3, 	axial pos 0, 	view = 0, 	tangential_pos_num = 0, 	timing position index = -1
{2, 0, 0}:0.000120942    ||   
{2, 1, 0}:0.000322843    ||   
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
CUT TO SIMPLIFY
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
{4, -28, 1}:5.42422e-09    ||   {4, -28, 1}:5.70331e-09
{4, -27, 0}:2.05508e-09    ||   {4, -27, 0}:2.159e-09
{4, -27, 1}:6.63398e-09    ||   {4, -27, 1}:6.97077e-09
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
CUT TO SIMPLIFY
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                       ||   {3, 32, -1}:0.125781
                       ||   {2, 0, 0}:0.000124743
                       ||   {2, 1, 0}:0.000332781
                       ||   {2, 2, 0}:0.000441099
                       ||   {2, 3, 0}:0.000581024
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
CUT TO SIMPLIFY
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```